kaolin/conversions/voxelgridconversions.py

```diff
@@ -336,11 +336,12 @@ def threshold(voxel: Union[torch.Tensor, VoxelGrid], thresh: float,
     """
     if isinstance(voxel, VoxelGrid):
         voxel = voxel.voxels
-    if not inplace:
         voxel = voxel.clone()
```

Comment:
then, shouldn't the attribute be named `voxel.voxelgrid` too ?

Reply:
the thing is that `voxel` is a `Union[torch.Tensor, VoxelGrid]`, which means it can be a voxelgrid or a voxel, which leads to this problem. Given that I'm not doing any modification on the naming part, I suggest that we fix that in another PR after a discussion.

Reply:
My reasoning was that we have a `Voxelgrid` class which contains a `voxels` object, so `voxelgrid.voxels` follows that naming scheme. That said, I see your point and agree we can move this discussion to another PR.